test/e2e-v2/cases/mongodb/otel-collector-config.yaml

```diff
@@ -0,0 +1,47 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+receivers:
+  prometheus:
+    config:
+     scrape_configs:
+       - job_name: 'mongodb-monitoring'
+         metrics_path: "/metrics"
+         scrape_interval: 10s
+         static_configs:
+           - targets: ['mongodb-exporter-1:9216','mongodb-exporter-2:9216']
+             labels:
+               cluster: replset
```

Comment:
This is hardcoded, can we use the real `replicaSet` name?

Reply:
"replset" value here just represent the cluster name. A mongodb cluster can have multi replsets, it likes shard in redis cluster. So we don't need the real replset name here.

Reply:
> "replset" value here just represent the cluster name. A mongodb cluster can have multi replsets, it likes shard in redis cluster. So we don't need the real replset name here.

This sounds unreasonable to me. Correct me if I am wrong, a MongoDB ReplicaSet has a name and it's composed of multiple replicas, the entire ReplicaSet is a `Service`, each of the replica is a `ServiceInstance` in SkyWalking concept, isn't it?

Reply:
> > "replset" value here just represent the cluster name. A mongodb cluster can have multi replsets, it likes shard in redis cluster. So we don't need the real replset name here.
> 
> This sounds unreasonable to me. Correct me if I am wrong, a MongoDB ReplicaSet has a name and it's composed of multiple replicas, the entire ReplicaSet is a `Service`, each of the replica is a `ServiceInstance` in SkyWalking concept, isn't it?

All ReplicaSet composed a mongodb cluster, which is a `Service`, it is right.
But each of the replica is  composed of multiple mongodb nodes(1 primary, others are secondary), and mongodb node is `ServiceInstance` in SkyWalking.

**mongodb-cluster**
![image](https://github.com/apache/skywalking/assets/16346644/99b396bc-c5f8-40ae-af92-cc8ad7c8062d)

**mongodb-replset**
![image](https://github.com/apache/skywalking/assets/16346644/5dd1ff1c-5c8d-4e40-8ee6-458af7ac2c88)